### PR TITLE
feat: Add integration provider for Holded

### PR DESCRIPTION
**Pull request by vrouet**
## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: packages/shared/providers.yaml [hunk 1]
```diff
@@ -1410,6 +1410,17 @@ highlevel-white-label:
     refresh_params:
         grant_type: refresh_token
     docs: https://docs.nango.dev/integrations/all/highlevel
+holded:
+    categories:
+        - accounting
```
Comment (hassan254-prog) [inline packages/shared/providers.yaml:1415]: Please add `holded` to the following individual categories in the documentation.

File: packages/shared/providers.yaml [hunk 2]
```diff
@@ -1410,6 +1410,17 @@ highlevel-white-label:
...
+        - crm
+        - invoicing
+    auth_mode: API_KEY
+    proxy:
```
Comment (hassan254-prog) [inline packages/shared/providers.yaml:1419]: With `API_KEY` and `BASIC` auth modes, it is advisable to test the implementation using the `proxy.verification` field when creating a new connection on Nango. You can have a look at the following [example](https://github.com/NangoHQ/nango/blob/1381cffd77d4b024011e994d595ef7dc2083222f/packages/shared/providers.yaml#L1675).

**Review by hassan254-prog — COMMENTED** [review]
@vrouet LGTM. I’ve left a few small comments inline 👍

File: packages/shared/providers.yaml (hunk 2 above)
Comment (vrouet) [reply packages/shared/providers.yaml:1419]: I noticed that, but these verification endpoint normally use an endpoint to get the current user, or something similar, and there is no such endpoint on Holded's API (see https://developers.holded.com/reference/api-key). What do you recommend?

File: packages/shared/providers.yaml (hunk 1 above)
Comment (vrouet) [reply packages/shared/providers.yaml:1415]: Done!

File: packages/shared/providers.yaml (hunk 2 above)
Comment (hassan254-prog) [reply packages/shared/providers.yaml:1419]: For consistency, you can replace the logo with , the [following](https://codefile.io/f/e3ZHAU34ny).

Comment (vrouet) [reply packages/shared/providers.yaml:1419]: I see that some of the other providers use a random list endpoint, so I've added GET `/invoicing/v1/contacts`, let me know if it looks ok to you

Comment (hassan254-prog) [reply packages/shared/providers.yaml:1419]: @vrouet , regarding the `proxy.verification` field, when creating a new connection: Nango checks if the status is `success` based on the API response status from the verification endpoint. This allows you to determine whether your implementation is correct. After adding this to the `providers.yaml` file, does creating a new connection work successfully?

Comment (vrouet) [reply packages/shared/providers.yaml:1419]: Yes, I did succeed in creating a connection on my local Nango with that provider config

Comment (vrouet) [reply packages/shared/providers.yaml:1419]: Logo updated :+1:

**Review by bodinsamuel — APPROVED** [review]
Thanks @hassan254-prog for taking care of the initial review.
Looks good on my end 👌🏻

**Review by hassan254-prog — APPROVED** [review]

**Comment by bodinsamuel** [comment]
thanks for your contribution @vrouet 🙌🏻 
It will be deployed during the week

**Pull request merged**